resteasy-spring-boot-starter-test/src/test/java/org/jboss/resteasy/springboot/LogbackTestApplicationListener.java

```diff
@@ -37,7 +39,10 @@ protected void append(ILoggingEvent event) {
                 return;
             }
             Level level = event.getLevel();
-            if ((level.equals(Level.WARN) || level.equals(Level.ERROR)) && !event.getMessage().startsWith(SCANNING_WARNING)) {
+            if ((level.equals(Level.WARN) || level.equals(Level.ERROR))
+                    && !event.getMessage().startsWith(SCANNING_WARNING)
+            && !event.getMessage().startsWith("InetAddress.getLocalHost")
```

Comment:
This is because under MacOS java1 sometimes the hostname resolving will print out warning.

Reply:
Maybe a comment in the code could help in the future

Reply:
okay!